content/news/036/index.md

```diff
@@ -88,6 +88,23 @@ include:
 [Tetra]: https://github.com/17cupsofcoffee/tetra
 [steamworks]: https://crates.io/crates/steamworks
 
+### [Robo Instructus: 3 Years Old][robo-3]
+
+![game logo + OS logos](../024/robo-os.jpg)
```

Comment:
We normally replicate images in each newsletter directory if it's needed, could you make a copy?